azurerm/internal/services/monitor/tests/data_source_monitor_scheduled_query_rules_alert_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
 )
 
-func TestAccDataSourceAzureRMMonitorScheduledQueryRules_alertingAction(t *testing.T) {
+func TestAccDataSourceAzureRMMonitorScheduledQueryRulesAlertingAction(t *testing.T) {
```

Comment:
Let's keep the `_` that was removed in. It lets us quickly see and grab the prefix when we want to run this subset of tests and has some other minor benefits that warrant keeping it in